microservicecloud/docker-compose.backup

```diff
@@ -1,4 +1,4 @@
-version: "3"
+ersion: "3"
```

Comment:
Small mistake here., Version name. But its benign. Will fix it, myself.
@TimothyHou

Reply:
thanks for pointing it out!